backend/src/Edna/Dashboard/DB/Query.hs

```diff
@@ -156,3 +205,12 @@ getRemovedMeasurements (SqlId subExperimentId) =
     removedMeasurement <- all_ $ esRemovedMeasurements ednaSchema
     guard_ (rmSubExperimentId removedMeasurement ==. val_ subExperimentId)
     pure removedMeasurement
+
+-- Helper
+getExperimentId :: SubExperimentId -> Edna Word32
+getExperimentId sqlId@(SqlId subExpId) = do
+  mRes <- runSelectReturningOne' $ select $ do
+    subExperiment <- all_ $ esSubExperiment ednaSchema
+    guard_ (seSubExperimentId subExperiment ==. val_ (SqlSerial subExpId))
+    pure (seExperimentId subExperiment)
+  justOrThrow (DESubExperimentNotFound sqlId) mRes
```

Comment:
So, again, it looks more convenient to throw errors from Service part and here return Maybe Word32